test/widget/tool/hand_tool_test.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:paintroid/service/device_service.dart';
+import 'package:paintroid/tool/tool.dart';
+import 'package:paintroid/ui/pocket_paint.dart';
+
+import '../utils/utils.dart';
```

Comment:
please always use the full import with package:, not the relative path

Reply:
I have looked at some other projects and they also use relative path and according to a post from [2020](https://stackoverflow.com/questions/64145591/how-to-import-test-util-files-inside-of-your-tests-on-dart-flutter) I believe this is not possible since the test folder is not a package. I think this can only be achieved only after we modularize the entire project. [Example](https://github.com/firebase/flutterfire/blob/master/tests/pubspec.yaml)